test/metric_value_benchmark/eks_daemonset_test.go

```diff
@@ -54,14 +61,42 @@ func (e *EKSDaemonTestRunner) validateInstanceMetrics(name string) status.TestRe
 			Value: dimension.UnknownDimensionValue(),
 		},
 	})
-
 	if len(failed) > 0 {
 		log.Println("failed to get dimensions")
 		return testResult
 	}
 
-	fetcher := metric.MetricValueFetcher{}
-	values, err := fetcher.Fetch("ContainerInsights", name, dims, metric.AVERAGE, metric.HighResolutionStatPeriod)
+	// get list of metrics that has more dimensions for container insights
+	// this is to avoid adding more dimension provider for non-trivial dimensions e.g. PodName
+	listFetcher := metric.MetricListFetcher{}
+	if slices.Contains(metricsWithMoreDimensions, name) {
+		metrics, err := listFetcher.Fetch(containerInsightsNamespace, name, dims)
```

Comment:
So dims is still just for `ClusterName` currently as per above. i.e. currently this test continues to only validate Type=Cluster log event generated metrics.
Will the other types be in future PRs?

Reply:
We could. The metric `pod_number_of_container_restarts` was chosen to take the other route since it's the only metric that doesn't have `Type=Cluster`. Is that one enough to be confident? I'm not sure, but it validates at least 1 non-cluster metric. Probably I should add each unique type metrics to take the other route as well.

Reply:
I think long term a schema match for each type of log event where we validate the presence of required dimensions/labels would be good. 
So if I understood the existing code, we are only validating a single log event of Type=Cluster